pymmcore/__init__.pyi

```diff
@@ -1014,10 +1014,22 @@ class CMMCore:
         """Sets the current slm device."""
     def setSLMExposure(self, slmLabel: str, exposure_ms: float) -> None:
         """For SLM devices with build-in light source (such as projectors),
-
         this will set the exposure time, but not (yet) start the illumination"""
+    @overload
+    def setSLMImage(self, slmLabel: str, pixels: np.ndarray[np.uint8]) -> None:
+        """
+        Write a 8-bit grayscale image to the SLM. Pixels must be a 2D numpy array [h,w] of uint8s.
+        Warning: SLM might convert grayscale to binary internally.
+        """
+    @overload
+    def setSLMImage(self, slmLabel: str, pixels: np.ndarray[np.uint8, np.uint8, np.uint8]) -> None:
```

Comment:
I don't think you can do `np.ndarray[np.uint8, np.uint8, np.uint8]` yet to declare shapes... (https://numpy.org/devdocs/reference/typing.html#d-arrays), so just use `npt.NDArray[np.uint8]` here as well